### PR TITLE
Publish 1.4 release

### PR DESCRIPTION
**Pull request by fossedihelm**
**What this PR does / why we need it**:

Publish 1.4 schedule

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by aburdenthehand** [comment]
/hold
Looks good to me and happy to lgtm. 
That said, if we push it back two weeks then we coincide with KubeCon NA and we can have it highlighted there and get extra attention on it. @fossedihelm @fabiand @rthallisey @vladikr thoughts?

**Comment by xpivarc** [comment]
> /hold Looks good to me and happy to lgtm. That said, if we push it back two weeks then we coincide with KubeCon NA and we can have it highlighted there and get extra attention on it. @fossedihelm @fabiand @rthallisey @vladikr thoughts?

Do you mean to push back feature freeze by 2 weeks?

**Comment by aburdenthehand** [comment]
Release date. Whether by extension that also includes feature freeze, I have no opinion.
Just a suggestion to consider - I know we already balance against quite a bit. It definitely helped drive attention to the project when we coincided release with KubeCon last year, but at this stage we're not sure what level of attendance from the project that we'll have this year.

**Comment by fabiand** [comment]
Delaying release date by 2 weeks for kubecon aligned sounds good to me.

Who is taking the action to then have press material ready for kubecon? :)

**Comment by fossedihelm** [comment]
@aburdenthehand @fabiand @xpivarc 
I pushed a [new proposal](https://github.com/kubevirt/sig-release/compare/bf133307ba7577948e098ee40a6cb6217974f0b5..76f8d0c2e16b6e25dd7b2a43bc92cf605e6bc82c).
I placed the release date exactly one day before the KubeCon NA, so that we could have attention starting from the first day. Please let me know if you prefer the 12th of November.
Thank you

File: releases/v1.4/schedule.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+| **September** |                    |                                |                                  |                   |
+| 2024-09-10    |                    |                                | CI lanes for provider are voting |                   |
+| 2024-09-17    | Beta 0             | Tag v1.4.0-beta.0              | Mandatory pre-submit             |                   |
+| **October**   |                    |                                |                                  |                   |
+| 2024-10-22    | Feature Freeze     | Branch release-1.4             |                                  |                   |
+| 2024-10-29    | RC 0               | Tag v1.4.0-rc.0 on release-1.4 |                                  |                   |
+| **November**  |                    |                                |                                  |                   |
+| 2024-11-04    | RC 1               | Tag v1.4.0-rc.1 on release-1.4 |                                  |                   |
+| 2024-11-11    | GA                 | Tag v1.4.0 on release-1.4      |                                  |                   |
```
Comment (aburdenthehand) [inline releases/v1.4/schedule.md:19]:
```suggestion
| 2024-11-05    | RC 1               | Tag v1.4.0-rc.1 on release-1.4 |                                  |                   |
| 2024-11-12    | GA                 | Tag v1.4.0 on release-1.4      |                                  |                   |
```
@fossedihelm I appreciate your thinking here but maintaining our Tuesday milestones and releasing on Day 1 of KubeCon is perfectly fine, and probably better from the cncf PR pov. Especially if the release is handled in the EMEA timezone and before everything kicks off. Thank you!

Comment (fossedihelm) [reply releases/v1.4/schedule.md:19]: Fine, thank you for the feedback

**Comment by aburdenthehand** [comment]
/lgtm
Grand. Thanks @fossedihelm !

**Pull request merged**